### PR TITLE
[currency][ecb importer] added base currency.

### PR DESCRIPTION
**Pull request by aitboudad**

### CODE REVIEW

File: src/Sylius/Component/Currency/Importer/EuropeanCentralBankImporter.php [hunk 1]
```diff
@@ -30,6 +30,9 @@ public function import(array $managedCurrencies = array())
     {
         $xml = @simplexml_load_file($this->url);
         if ($xml instanceof \SimpleXMLElement) {
+            // base currency: euro
+            $this->updateOrCreate($managedCurrencies, 'EUR', 1.00);
```
Comment (aRn0D) [inline src/Sylius/Component/Currency/Importer/EuropeanCentralBankImporter.php]: Perhaps you should not hard code 'EUR'

Comment (pjedrzejewski) [inline src/Sylius/Component/Currency/Importer/EuropeanCentralBankImporter.php]: This. We should use the parameter.

Comment (aitboudad) [inline src/Sylius/Component/Currency/Importer/EuropeanCentralBankImporter.php]: ok I'll update it

Comment (aitboudad) [inline src/Sylius/Component/Currency/Importer/EuropeanCentralBankImporter.php]: done !

**Comment by aRn0D** [comment]
@aitboudad I think you should squash your commit :)

**Comment by aitboudad** [comment]
@albanR done, I found an issue for eor importer see #2581

**Comment by stloyd** [comment]
@pjedrzejewski Seems good :+1:

**Comment by stloyd** [comment]
@pjedrzejewski @aRn0D ping...

**Comment by pjedrzejewski** [comment]
Thank you @aitboudad! :+1:

**Pull request merged**